parlai/utils/conversations.py

```diff
@@ -64,6 +66,10 @@ def _get_path(datapath):
         fle, _ = os.path.splitext(datapath)
         return fle + '.metadata'
```

Comment:
nit: i recently found a bug where some code that was using splitext (bytelevelbpe) wasn't working properly when I failed to give a file extension (which happened a lot there). It's just a potential red flag